upload/upload.go

```diff
@@ -22,15 +23,15 @@ import (
 // GetFile verifies that the proper upload field is in place and returns the file
 func GetFile(r *http.Request) (multipart.File, *multipart.FileHeader, error) {
 	var err error
```

Comment:
this doesn't seem to be used anymore now

Reply:
fixed, thanks!